### ISSUE TITLE
MSFT-74OAuth provider links not accessible via Tab

### ISSUE THREAD

**Issue by veshu** [veshu/testjiraimport#98]
**Steps to Reproduce**
1. Go to https://accounts.topcoder.com/member
2. Open VoiceOver on Mac.
3. Press tab till you go see the SSO highlight.

**Expected Result**
The other OAuth providers should be accessible on pressing Tab.

**Actual Result**
The other OAuth providers except SSO are not accessible via Tab.

**Device**: Desktop/Labtop
**Operating System**: macOS
**Browser**: Chrome
**WCAG Category**: 2. Operable
**WCAG Level**: A
**WCAG Success Criterion**: 2.4.3 Focus Order

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=1f11c1b0%2Dabf6%2D41bc%2Dbbc3%2D8435a17fcc67&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2Fanmolwassan%2FBug%202
